zephyr/ra/ra_cfg/fsp_cfg/r_mipi_dsi_cfg.h

```diff
@@ -0,0 +1,18 @@
+/*
+* Copyright (c) 2020 - 2024 Renesas Electronics Corporation and/or its affiliates
+*
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#ifndef R_MIPI_DSI_CFG_H_
+#define R_MIPI_DSI_CFG_H_
+#ifdef __cplusplus
+            extern "C" {
+            #endif
```

Comment:
@thenguyenyf : Please unify format with other files.

Reply:
I fixed it